tasklist/els-schema/src/main/java/io/camunda/tasklist/schema/IndexMapping.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.tasklist.schema;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.tasklist.exceptions.TasklistRuntimeException;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class IndexMapping {
+
+  private String indexName;
+
+  private String dynamic;
+
+  private Set<IndexMappingProperty> properties;
+
+  private Map<String, Object> metaProperties;
+
+  public String getIndexName() {
+    return indexName;
+  }
+
+  public IndexMapping setIndexName(final String indexName) {
+    this.indexName = indexName;
+    return this;
+  }
+
+  public String getDynamic() {
+    // Opensearch changes the capitalization of this field on some query results, change to
+    // lowercase for consistency
+    return dynamic == null ? null : dynamic.toLowerCase();
```

Comment:
lowercasing is already handled in the setter, this is redundant, isn't?

Reply:
Good catch :)